### ISSUE TITLE
Server Slot Request...

### ISSUE THREAD

**Issue by Dausk03** [djohn49/sgc_issue_tracker#11]
I believe either you or Connie mentioned that there was a spare server slot. I was wondering if we could use the slot for a Pixelmon server, since a couple of people who logged in today were supportive of the idea. Thank you for considering it.

**Comment by djohn49** [comment]
Yes, it can be used for that. Someone else needs to make the pack, though. Once that happens I have no problem deploying it.
